tests/IndexTest.php

```diff
@@ -626,17 +688,19 @@ public function testCreateWithInvalidOption(): void
         $this->expectException(InvalidException::class);
         $this->expectExceptionMessageMatches('/"testing_invalid_option" is not a valid option\. Allowed options are ((("[a-z_]+")(, )?)+)\./');
 
-        $client = $this->_getClient();
-        $indexName = 'test';
-        $index = $client->getIndex($indexName);
+        $client = $this->createMock(Client::class);
+        $index = new Index($client, 'test');
 
         $opts = ['testing_invalid_option' => true];
         $index->create([], $opts);
     }
 
+    /**
+     * @group unit
```

Comment:
unit?

Reply:
Yes unit as there is no call done on the elasticsearch stack. Mocking a client is sufficient as it's not used for this test